recipes/fmt/all/test_cmakedeps/conanfile.py

```diff
@@ -0,0 +1,25 @@
+import os
+from conans import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps
+from conan import tools
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "CMakeDeps"
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["FMT_HEADER_ONLY"] = self.options["fmt"].header_only
+        tc.generate()
+
+    def build(self):
+        cwd = os.path.dirname(__file__)
```

Comment:
No usage of ``__file__`` should be necessary. You might want to try defining a ``layout()``.

Reply:
still, it would be something like this if I want to use the absolute folder, right?:

```python
def layout(self):
   cmake_layout(self)
   cwd = os.path.dirname(__file__)
   self.folders.source = os.path.join(cwd, "..", "test_package")
```

Anyway, I will change it to a relative folder